CHANGELOG.md

```diff
@@ -5465,3 +5466,5 @@ Released 2018-09-13
 [`accept-comment-above-attributes`]: https://doc.rust-lang.org/clippy/lint_configuration.html#accept-comment-above-attributes
 [`allow-one-hash-in-raw-strings`]: https://doc.rust-lang.org/clippy/lint_configuration.html#allow-one-hash-in-raw-strings
 <!-- end autogenerated links to configuration documentation -->
+onfiguration.html#allow-private-error
```

Comment:
Looks like something went wrong here.

Reply:
I'm not sure why that keeps happening :confused: Will probably need to keep a closer eye on the changelog after rebasing